aten/src/ATen/native/UpSampleNearest2d.cpp

```diff
@@ -58,73 +16,92 @@ static void upsample_nearest2d_backward_out_cpu_template(
       "It is expected input_size equals to 4, but got size ",
       input_size.size());
 
-  int64_t output_height = output_size[0];
-  int64_t output_width = output_size[1];
+  int64_t output_width = output_size[0];
+  int64_t output_height = output_size[1];
```

Comment:
This also looks wrong

Reply:
Looks wrong and not caught by test - can you please add a test with the different input and output width and height?

Reply:
I think the code here is functionally correct; just the error message would be rendered wrong here.